### PR TITLE
Added support for LinkedList<> type

### PR DESCRIPTION
**Pull request by YPermitin**
#37 Added support for LinkedList<> type + unit tests. Added unit tests for TypeHelper.

### CODE REVIEW

File: src/BinaryFormatter/Utils/TypeHelper.cs [hunk 1]
```diff
@@ -24,5 +26,45 @@ public static class TypeHelper
...
+        public static bool IsLinkedList(object obj)
+        {
+            Type[] genericTypes = obj.GetType().GenericTypeArguments;
+
+            if (genericTypes.Length == 1)
+            {
+                var listType = typeof(LinkedList<>);
+                var constructedListType = listType.MakeGenericType(genericTypes);
+                return HasConversionOperator(obj.GetType(), constructedListType);
+            } else
```
Comment (lukasz-pyrzyk) [inline src/BinaryFormatter/Utils/TypeHelper.cs]: i guess else statement is redundant

Comment (YPermitin) [reply src/BinaryFormatter/Utils/TypeHelper.cs]: @lukasz-pyrzyk Agree. FIx in the commit 11163ed

**Comment by lukasz-pyrzyk** [comment]
Please update ```readme``` and I'm happy to merge and release 2.1 :)

**Comment by YPermitin** [comment]
@lukasz-pyrzyk I did it.

Added info about LinkedList and other to the "collection" part.
Also I added info about not supported types.

Is it ok?

**Pull request merged**